src/main/java/core/basesyntax/service/file/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.file;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderServiceImpl implements FileReaderService {
+
+    @Override
+    public List<String> read(String path) {
+        try {
+            return Files.readAllLines(Path.of(path));
+        } catch (IOException e) {
+            throw new RuntimeException("Cannot read file: " + path);
```

Comment:
Consider including the original `IOException` as a cause in the `RuntimeException` to provide more context for debugging. For example: `throw new RuntimeException("Cannot read file: " + path, e);`